cni/network/network.go

```diff
@@ -1404,3 +1427,14 @@ func convertCniResultToInterfaceInfo(result *cniTypesCurr.Result) network.Interf
 
 	return interfaceInfo
 }
+
+func findDefaultInterface(ipamAddResult IPAMAddResult) int {
+	for i, ifInfo := range ipamAddResult.interfaceInfo {
+		if ifInfo.NICType == cns.InfraNIC {
+			return i
+		}
+	}
+
+	// error or nil here
```

Comment:
And a log as well either way.

Reply:
added an error. Don't see a need to add a CNI log here as we capture the `InfraNIC` in multiple ways.